classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt

```diff
@@ -0,0 +1,41 @@
+### Exercise 5.1 - Docker
+Ran the existing command from my inital containers created in Exercise 2 but pulled it from my Docker Hub repoistry 
+I ensued that I was logged into Docker CLI and ran the following
+
+
+##Pulling images
+1 = docker pull starkmatt/exercise02:v1
+2 = docker pull starkmatt/exercise02:v2
+
+
+➜  starkmatt git:(starkmatt/c03-docker05) ✗ docker pull starkmatt/exercise02:v1
```

Comment:
it;s fine to pull the images before, but docker will realise it doesn't have it locally and will download when you execute the `docker run` command.

Reply:
I see,
Thanks!